### PR TITLE
fix(platform): fixing accessibility of multi combobox component

### PR DESCRIPTION
**Pull request by web-dev23**
## Related Issue(s)

Closes https://github.com/SAP/fundamental-ngx/issues/5766

## Description
Fix of accessibility issues of Multi Combobox component.

##### During Implementation

1. Visual Testing:

-   [x] visual misalignments/updates
-   [x] check Light/Dark/HCB/HCW themes
-   [x] RTL/LTR - proper rendering and labeling
-   [x] responsiveness(resize)
-   [x] Content Density (Cozy/Compact/(Condensed))
-   [x] States - hover/disabled/focused/active/on click/selected/selected hover/press state
-   [x] Interaction/Animation - open/close, expand/collapse, add/remove, check/uncheck
-   [x] Mouse vs. Keyboard support
-   [x] Text Truncation

2. API and functional correctness

-   [x] check for console logs (warnings, errors)
-   [x] API boundary values
-   [x] different combinations of components - free style
-   [x] change the API values during testing

3. Documentation and Example validations

-   [x] missing API documentation or it is not understandable
-   [x] poor examples
-   [x] Stackblitz works for all examples

4. Accessibility testing
5. Browser Testing - Edge, Safari, Chrome, Firefox

##### PR Quality

-   [x] the commit message(s) follows the guideline:
        https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
-   [x] tests for the changes that have been done
-   [x] all items on the PR Review Checklist are addressed :
        https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
-   [x] Run npm run build-pack-library and test in external application
-   [x] update `README.md`
-   [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)

### CODE REVIEW

File: libs/platform/src/lib/form/multi-combobox/multi-combobox/multi-combobox.component.html [hunk 1]
```diff
@@ -292,6 +296,7 @@
     <fd-form-message
         *ngIf="isSearchInvalid"
         type="error"
+        [id]="status"
```
Comment (dimamarksman) [inline libs/platform/src/lib/form/multi-combobox/multi-combobox/multi-combobox.component.html:299]: Doesn't look like a unique id.
From where does this `status` come from?

File: libs/platform/src/lib/form/multi-combobox/multi-combobox/multi-combobox.component.html [hunk 2]
```diff
@@ -88,6 +89,7 @@
                 [(ngModel)]="inputText"
                 (ngModelChange)="searchTermChanged()"
                 [placeholder]="placeholder"
+                [title]="placeholder"
```
Comment (dimamarksman) [inline libs/platform/src/lib/form/multi-combobox/multi-combobox/multi-combobox.component.html:92]: are you sure you need to add a title to the input text?

File: libs/platform/src/lib/form/multi-combobox/multi-combobox/multi-combobox.component.html [hunk 3]
```diff
@@ -71,6 +71,7 @@
                 type="text"
                 role="combobox"
                 autocomplete="off"
+                [aria-describedby]="status"
```
Comment (dimamarksman) [inline libs/platform/src/lib/form/multi-combobox/multi-combobox/multi-combobox.component.html:74]: this just reads the status itself how it is (e.x. `warning`) it doesn't say that this info is about a control state (e.x. Value State Warning)

**Review by dimamarksman — COMMENTED** [review]
Please see my comments.

The issue itself consists of 18 points but here I don't see many changes.
Are all others points are not reproducible? If so it would be good to point it out here.

**Comment by web-dev23** [comment]
Issue 1 - fixed
Issue 2 - fixed
Issue 3 - not possible to move from first item to last using arrow up and from last item to first using arrow down - expected behavior, same is on combobox component
Issue 4 - fixed
Issue 5 - unable to replicate
Issue 6 - unable to replicate
Issue 7 - unable to replicate
Issue 8 - fixed horizontal scroll, added alt text, regarding tokens are not clearly visible to the user - expected behavior
Issue 9 - fixed
Issue 10 - expected bahavior, the same way behaves combobox component.
Issue 11 - fixed
Issue 12 - unable to replicate
Issue 13 - when navigating on multi select token label are read by screen reader
Issue 14 - unable to replicate
Issue 15 - fixed item number, when navigating the tokens within multi select the information that token is deletable is read
Issue 16 - unable to replicate
Issue 17 - fixed
Issue 18 - unable to replicate

File: libs/platform/src/lib/form/multi-combobox/multi-combobox/multi-combobox.component.html (hunk 3 above)
Comment (Betrozov) [reply libs/platform/src/lib/form/multi-combobox/multi-combobox/multi-combobox.component.html:74]: also `[aria-describedby]` -> `[attr.aria-describedby]`

**Pull request closed without merging**